Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/SamplePaywallsList.swift

```diff
@@ -87,8 +90,10 @@ struct SamplePaywallsList: View {
                     introEligibility: Self.introEligibility
                 )
             )
-        case .customerCenter:
+        case .customerCenterSheet:
             CustomerCenterView(customerCenterActionHandler: self.handleCustomerCenterAction)
```

Comment:
Actually, this needs to be `EmptyView` as well because view is handled by the view modifier